CHANGELOG.md

```diff
@@ -2,8 +2,8 @@
 
 ## Unreleased
 
-TBD
-
+### Features
+- [\#32](https://github.com/medibloc/panacea-java/pull/32) feat: Upgrade `Stargate` version  
```

Comment:
How about modifying this to `Make compatible with Panacea v2.0.0 based on Cosmos SDK Stargate`?

Reply:
Thank you!
I changed that